NeuralAmpModeler/NeuralAmpModeler.cpp

```diff
@@ -256,7 +256,12 @@ NeuralAmpModeler::NeuralAmpModeler(const InstanceInfo& info)
       pControl->SetMouseOverWhenDisabled(true);
     });
 
-    pGraphics->GetControlWithTag(kCtrlTagOutNorm)->SetMouseEventsWhenDisabled(false);
+    pGraphics->GetControlWithParamIdx(kOutNorm)->SetMouseEventsWhenDisabled(false);
+    pGraphics->GetControlWithParamIdx(kNoiseGateActive)->SetMouseEventsWhenDisabled(false);
+    pGraphics->GetControlWithParamIdx(kEQActive)->SetMouseEventsWhenDisabled(false);
+    pGraphics->GetControlWithParamIdx(kIRToggle)->SetMouseEventsWhenDisabled(false);
+
+    pGraphics->GetControlWithParamIdx(kNamToggle)->SetValueFromUserInput(1); // enabled by default
```

Comment:
[Critical] Instead of doing it like this, please initialize the parameter for `kNamToggle` up top around line 73.

If you run the unit tests provided by Steinberg, you may see a failure due to this. Can you help me with reviewing this PR by running them and ensuring that everything passes?

Also, can you ensure that if you save the plug-in in your session on both states, that it comes back up in the state that it was saved in when you restart your DAW?

Reply:
Indeed UT failed due to this. Fixed in latest version.

About plugin state restore in DAW, yes - now it works, but requires plugin version update before, to properly unserialize current state instead of "legacy".